device/edison.go

```diff
@@ -99,48 +100,70 @@ func setConfig() error {
 	// get IP
 	var ip string
 
-	i := dialogs.SelectOneDialog("Chose the edison's inteface to connect via usb: ", []string{"Default", "Enter IP"})
+	i := dialogs.SelectOneDialog("Choose Edison's interface to connect to: ", []string{"Select from a list of interfaces", "Enter IP manually"})
 	fallback := false
 
 	if i == 0 {
 		ifaces, err := help.LocalIfaces()
 
 		if err != nil || len(ifaces) == 0 {
-			logrus.Error(err)
+			log.Error(err)
 			fmt.Println("[-] ", err.Error())
 			fallback = true
 		}
 
 		if !fallback {
 			arr := make([]string, len(ifaces))
 			arrSel := make([]string, len(ifaces))
+			fmt.Println("[+] Highlighted interfaces is our heuristic guess")
 			for i, iface := range ifaces {
 				arr[i] = iface.Name
 				arrSel[i] = iface.Name
 				if iface.Ipv4[:4] == "169." {
-					arrSel[i] = "\x1b[34m" + iface.Name + "\x1b[0m"
-					fmt.Println("[+] The highlighted interface is our heuristic guess")
+					if runtime.GOOS == "windows" {
```

Comment:
__[gometalinter]__
* [goconst] 2 other occurrence(s) of "windows" found in: edison.go:132:23 edison.go:155:22

<!-- Comment by [SideCI](https://sideci.com) -->